src/components/LearnerActivityTable/__snapshots__/LearnerActivityTable.test.jsx.snap

```diff
@@ -265,7 +265,7 @@ exports[`LearnerActivityTable renders active learners table correctly 1`] = `
                 <td
                   className=""
                 >
-                  October 21, 2017
+                  October 22, 2017
```

Comment:
Did you get a chance to know why this happens ?

Reply:
I guess the snapshots are generated on current date basis so thats why they get changed.

Reply:
Can you remove your snapshot and/or may be not push it. Yesterday I created a brand new branch off master and it worked perfectly fine. Also, may be you need to rebase with master?